### PR TITLE
Updated to HTML5 standards

### PR DESCRIPTION
**Pull request by JonMacRae**
I noticed when I validated my updates there weren't any errors when I used the direct input and also when I loaded the file, however when I tried to use the URL of the update in my repo it set off numerous errors.  Is that to be expected, if so can you explain why?

### CODE REVIEW

**Comment by mbMosman** [comment]
I am guessing that if you linked it to the GitHub page where you can see the source, that there might be issues with other content surrounding your HTML?  

Did you use this URL:  http://jonmacrae.github.io/validation-lab/ in the validator?  That is to you page hosted live on the web (get it from Settings).

**Comment by JonMacRae** [comment]
I did not, I used the URL at the top of the page. So when you validate via URL you are validating all .html files in that repo?

**Comment by mbMosman** [comment]
No, when you give the W3C validator a URL, it is just validating literally that page, so if you were to right click on the page and say View Source, that is what you are validating, not your index.html file itself.

**Comment by JonMacRae** [comment]
Ahhh I see, alright that makes more sense. 
Thanks!

File: index.html [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 <body>
-    <title>Fix the Errors<title>
     <main>
-    <h2>Some Heading</h>
-    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-    </main>
-    <main>
-    <h2>Some Other Heading</h>
-    <p>Vestibulum sollicitudin leo a enim imperdiet, sed vehicula lectus varius. Donec vitae ex et elit efficitur sollicitudin a ac justo. Morbi volutpat enim vitae erat porttitor bibendum. Vestibulum maximus commodo libero, fermentum viverra arcu tincidunt id.
+    <h1>Fix the Errors</h1>
```
Comment (mbMosman) [inline index.html]: You might try to keep the `<header>` around the `<h1>`.  
Make sure that you understand that the `<head>` and the `<header>` are different things.  This is often a point of confusion.

**Comment by mbMosman** [comment]
Nice work!

**Pull request closed without merging**